### PR TITLE
[niina] Append `m_` to all member variables of COMAI_HI class

### PR DESCRIPTION
**Pull request by peria**
It is difficult to judge if a variable is local or not.
So let me append `m_` to all member variables.

### CODE REVIEW

File: src/cpu/test_lockit/coma.cc [hunk 1]
```diff
@@ -239,7 +239,7 @@ int COMAI_HI::aite_hyouka(const int ba3[6][kHeight], int tsumo[])
     int setti_basyo[4];
     int dabuchk[20];
     int ichiren_kesi;
-    int score_aa = 0;
+    int m_score_aa = 0;
```
Comment (mayah) [inline src/cpu/test_lockit/coma.cc]: not a member.

File: src/cpu/test_lockit/coma.cc [hunk 2]
```diff
@@ -470,7 +470,7 @@ int COMAI_HI::hyouka(const int ba3[6][kHeight], int tsumo[], int zenkesi_own, in
     int zenkes[22][22][22] {};
     int zenke[22] {};
     int setti_basyo[4];
-    int myf_kosuu_kesi = 0, myf_kosuu_iro = 0;
+    int m_myf_kosuu_kesi = 0, m_myf_kosuu_iro = 0;
```
Comment (mayah) [inline src/cpu/test_lockit/coma.cc]: not a member.

File: src/cpu/test_lockit/coma.cc [hunk 3]
```diff
@@ -2684,7 +2684,7 @@ int COMAI_HI::pre_hyouka(const int ba3[6][kHeight], int tsumo[], int zenkesi_own
     int zenkes[22][22][22] {};
     int zenke[22] = { 0 };
     int setti_basyo[4];
-    int myf_kosuu_kesi = 0, myf_kosuu_iro = 0;
+    int m_myf_kosuu_kesi = 0, m_myf_kosuu_iro = 0;
```
Comment (mayah) [inline src/cpu/test_lockit/coma.cc]: not a member.

**Comment by mayah** [comment]
Could you squash these commits? Not a good habit to leave the commits that don't compile.

**Comment by peria** [comment]
Let me close this PR and make another one.
github is difficult...

**Pull request closed without merging**